src/main/java/duke/CreateFileDir.java

```diff
@@ -0,0 +1,24 @@
+package duke;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class CreateFileDir {
+    public static void CreateFile() {
+        try {
+            Files.createDirectories(Paths.get("D:\\Documents\\Modules\\Sem 03\\CS2113\\iP\\data"));
+            File myObj = new File("D:\\Documents\\Modules\\Sem 03\\CS2113\\iP\\data\\duke.txt"); //D:\Documents\Modules\Sem 03\CS2113\iP\data
```

Comment:
Might not work on other systems with no D drive.
This won't work in non-Windows machine because of different path naming convention.
You may consider using relative path instead of absolute path.